### ISSUE TITLE
min-height for the img element?

### ISSUE THREAD

**Issue by LittleBigThing** [WordPress/twentynineteen#105]
Currently, there is a `min-height` set for the `img` element using SASS:

```
img {
    min-height: calc(3 * #{$size__spacing-unit});
}
```

I think there are some problems here:
- it is not clear for me why a min-height is needed
- it should probably not use `calc()` as it results in `calc(3 * 1rem)`in CSS instead of 3rem what is probably meant to happen
- it affects emojis as well:

<img width="624" alt="screenshot 2018-10-17 at 08 51 29" src="https://user-images.githubusercontent.com/5901923/47067979-5a75f600-d1eb-11e8-9481-1c076ce9f2d6.png">

So if we keep it, we should set it as

```
img {
    &:not(.emoji),
    &:not(.wp-smiley) {
        min-height: 3 * #{$size__spacing-unit};
    }    
}
```

<img width="623" alt="screenshot 2018-10-17 at 08 51 46" src="https://user-images.githubusercontent.com/5901923/47068173-e12ad300-d1eb-11e8-9caa-1e190975963e.png">

**Comment by LittleBigThing** [comment]
`calc()` is meant to calculate a mathematical expression in CSS, but mostly useful if the calculation involves different units. For example, `calc(100% - 20px)`. The above is not really a useful expression, except when the variable is a CSS custom property.

**Comment by kjellr** [comment]
Fixed via #106. The `calc()` in question was removed, so we can table that for now. 🙂
